packages/web/src/react.test.tsx

```diff
@@ -46,4 +46,82 @@ describe('<Analytics />', () => {
       expect(script).toHaveAttribute('defer');
     });
   });
+
+  describe('track custom events', () => {
```

Comment:
Just noticed that we're missing two describe blocks for `'in development mode'` and `'in production mode'` to test the separate modes

Reply:
did it once in generic. not needed to just copy it